Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -327,7 +327,8 @@ def run_search(search_object, query, scope=None, size=None, sort=None, order=Non
     status = 'in progress' if in_progress else 'complete'
     search_info = {
         'AFCookie': result.get('af_cookie'),
-        'Status': status
+        'Status': status,
+        'SessionStart': datetime.now().strftime("%m/%d/%Y, " "%H:%M:%S")
```

Comment:
* future additions will show up as single line change and not two
* follow up after demisto date format (ISO format without millisec)
```suggestion
        'SessionStart': datetime.now().strftime("%Y-%m-%dT%H:%M:%S"),
```

Reply:
I  haven't submitted the PB yet because I checked the changes through a copy of the integration and saw that the 'SessionStart' field I added causes the level bug in demisto so I need to change the extend context once again.